models/user.rb

```diff
@@ -10,9 +10,13 @@ class User < Base
 
   RESET_WINDOW = 60 * 15 # 15 minutes
 
-  SETTINGS = %w[
-    notifications bg font bg2 font2 red_logo white yellow green brown gray red blue
-  ].freeze
+  opts = %w[notifications red_logo bg font bg2 font2 white yellow green brown gray red blue]
```

Comment:
just do SETTINGS = [].map

Reply:
?

Reply:
rewrite this to do it all inline

using the form SETTINGS = %w[...].map

Reply:
hm, but I don’t want to map these settings, they are ok as is

Reply:
what? you don't use opts anywhere else, it's used to populate settings, instead of creating opts and mutating it, create SETTINGS in a functional style